docs/products/kafka/howto/rotate-credentials.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Rotate credentials for an Apache Kafka subscription
+sidebar_label: Rotate credentials
+---
+
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+import RelatedPages from "@site/src/components/RelatedPages";
+
+Rotate credentials for an Apache Kafka subscription to replace outdated credentials and keep access secure and approved.
+The request must be approved by another member of the owner group before new
+credentials can be downloaded.
+
+## Prerequisites
+
+You must be a member of the **owner group** of the access to rotate.
```

Comment:
Move it to the **Limitations** section? See my next comment.

And ow about other prerequisites? Are there any? Access to the console? a running service? not sure what specifically.

Reply:
I'd add the **Limitations** section:

- You must be a member of the owner group of the access to rotate credentials.
- Another member of the owner group must approve the request before new credentials can be downloaded.
- Only one active rotation request is allowed per access at a time.
- Credentials can only be viewed once. Make sure to save them securely.

With that, I'd remove the items above from all the other locations in the doc, not to duplicate this info.

Reply:
Thanks for the suggestions! I see your point, but these are expected system behaviors rather than limitations. Including them inline with the relevant actions helps clarify the process for readers, for example, understanding when a rotation request can be fulfilled. This approach provides immediate context and improves the user's comprehension of the steps involved. So, I would like to keep them in context rather than moving them to a 'Limitations' section, which wouldn't make sense here.

Additionally, regarding your question about other prerequisites, a running Kafka service is not strictly necessary for the rotation itself. The rotation process is tied to the governance access and the associated service user, independent of the service's operational status.